templates/py-input/main.py

```diff
@@ -0,0 +1,5 @@
+from scramjet.streams import Stream
```

Comment:
same as TS. Py examples should be same as those in js, so if we use streams in py than js shoud use streams, if we use generators js should use generators etc.